### PR TITLE
[DOCS-5328] Add beta docs for Oracle

### PR DESCRIPTION
**Pull request by jmeunier28**
### What does this PR do?

Adds private beta documentation for Oracle active sessions 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by estherk15 — COMMENTED** [review]
Created an editorial review card for this! https://datadoghq.atlassian.net/browse/DOCS-5328

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  text: "Basic Oracle Integration"
+
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">
+The features described on this page are in beta. Contact your Customer Success Manager to get the required agent build and installation directions.
```
Comment (caseycull) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:17]: capitalize Agent

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+1. [Grant the Agent access to the database](#grant-the-agent-access)
+2. [Install the Agent](#install-the-agent)
+
+## Before you begin
+
+Supported Oracle versions
+: 19c, 21c
+
+Supported Agent versions
+: Please contact your Customer Success Manager to get agent installation directions for the beta.
```
Comment (caseycull) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:33]: capitalize Agent

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+grant create session to c##datadog ;
+grant select on v_$session to c##datadog ;
+grant select on v_$database to c##datadog ;
+grant select on v_$containers to c##datadog;
+grant select on v_$sqlstats to c##datadog ;
+grant select on v_$instance to c##datadog ;
+grant select on dba_feature_usage_statistics to c##datadog ;
+```
+
+Create a new `view`, and give the agent user access to it:
```
Comment (caseycull) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:55]: capitalize Agent

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+  AND s.ksusepsi = sq_prev.sql_id(+)
+  AND s.ksusepha = sq_prev.plan_hash_value(+)
+  AND s.con_id = c.con_id(+)
+  AND s.ksuudoct = comm.command_type(+)
+;
+
+GRANT SELECT ON dd_session TO c##datadog ;
+```
+
+Grant additional permissions if you wish to use the basic oracle integration as well:
```
Comment (caseycull) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:148]: capitalize Oracle

**Review by caseycull — CHANGES_REQUESTED** [review]

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
+---
+title: Setting Up Database Monitoring for self-hosted Oracle
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:2]:
```suggestion
title: Setting Up Database Monitoring for Self-Hosted Oracle
```

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+---
+title: Setting up Oracle
+kind: documentation
+description: Setting up Database Monitoring on an Oracle database
+disable_sidebar: true
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/_index.md:5]:
```suggestion
disable_sidebar: true
private: true
is_beta: true
```

Comment (drichards-87) [reply content/en/database_monitoring/setup_oracle/_index.md:5]: Adding the above front matter will hid the article from appearing in search results. This is standard for private betas, although let me know if you'd prefer that it be discoverable via search.

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">
+The features described on this page are in beta. Contact your Customer Success Manager to get the required agent build and installation directions.
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/_index.md:13]:
```suggestion
The features described on this page are in private beta. Contact your Customer Success Manager for information on the required Agent build and installation instructions.
```

File: content/en/database_monitoring/setup_oracle/selfhosted.md (hunk 1 above)
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:17]:
```suggestion
The features described on this page are in private beta. Contact your Customer Success Manager for information on the required Agent build and installation instructions.
```

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">
+The features described on this page are in beta. Contact your Customer Success Manager to get the required agent build and installation directions.
+</div>
+
+Database Monitoring provides deep visibility into your Oracle databases by exposing query samples to profile your different workloads and diagnose issues quickly.
+
+Do the following steps to enable Database Monitoring with your database:
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:22]:
```suggestion
Complete the following steps to enable Database Monitoring with your database:
```

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+
+1. [Grant the Agent access to the database](#grant-the-agent-access)
+2. [Install the Agent](#install-the-agent)
+
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:26]:
```suggestion
1. [Grant the Agent access to the database](#grant-the-agent-access).
2. [Install the Agent](#install-the-agent).

```

File: content/en/database_monitoring/setup_oracle/selfhosted.md (hunk 2 above)
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:33]:
```suggestion
For more information on the required Agent build, contact your Customer Success Manager.
```

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+    server: '<HOSTNAME>:<SQL_PORT>'
+    service_name: "<SERVICE_NAME>" # The Oracle CDB service name
+    username: 'c##datadog'
+    password: '<PASSWORD>'
+    tags:  # Optional
+      - 'service:<CUSTOM_SERVICE>'
+      - 'env:<CUSTOM_ENV>'
+```
+
+Use the `service` and `env` tags to link your database telemetry to other telemetry through a common tagging scheme. See [Unified Service Tagging][3] on how these tags are used throughout Datadog.
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:176]:
```suggestion
Use the `service` and `env` tags to link your database telemetry to other telemetry through a common tagging scheme. See [Unified Service Tagging][3] to learn more about how these tags are used in Datadog.
```

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+
+Once all Agent configuration is complete, [restart the Datadog Agent][4].
+
+To install the basic Oracle integration, you should create an additional Oracle Agent conf file at `/etc/datadog-agent/conf.d/oracle.d/conf.yaml`
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:180]:
```suggestion
To install the basic Oracle integration, you should create an additional Oracle Agent conf file at `/etc/datadog-agent/conf.d/oracle.d/conf.yaml`.
```

Comment (drichards-87) [reply content/en/database_monitoring/setup_oracle/selfhosted.md:180]: I'm not too familiar with our Oracle integration(s). Do we have more than one? If not, we can probably remove the word "basic".

File: content/en/database_monitoring/setup_oracle/selfhosted.md [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+```SQL
+GRANT CREATE SESSION TO c##datadog CONTAINER=ALL;
+Grant select any dictionary to c##datadog container=all;
+GRANT SELECT ON GV_$PROCESS TO c##datadog CONTAINER=ALL;
+GRANT SELECT ON gv_$sysmetric TO c##datadog CONTAINER=ALL;
+```
+
+## Install the Agent
+
+To start collecting Oracle telemetry, first [install the Datadog Agent][1]. In order to install the correct version to participate in the beta, please contact your Customer Success Manager.
```
Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/selfhosted.md:159]:
```suggestion
To start collecting Oracle telemetry, first [install the Datadog Agent][1]. Note that you must install the correct version to participate in the beta. Contact your Customer Success Manager for more information.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some feedback from the Docs team.

File: content/en/database_monitoring/setup_oracle/selfhosted.md (hunk 11 above)
Comment (jmeunier28) [reply content/en/database_monitoring/setup_oracle/selfhosted.md:180]: yeah there is a regular oracle integration and this new one that we have written (eventually they will be merged), and they can be thought of as different feature sets

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+|  | Self-hosted |
+|--|------------|
+| Oracle 19c | {{< X >}} |
+| Oracle 21c | {{< X >}} |
+
+The following deployment configurations are supported: on-prem, single-node, multi-tenant. Note these do not include RAC and Exadata or legacy architectures.
+
+For setup instructions, select your hosting type:
+
+{{< partial name="dbm/dbm-setup-oracle" >}}
```
[the remark this answers is not in the document]
Comment (jmeunier28) [reply content/en/database_monitoring/setup_oracle/_index.md:28]: yes, we will support many more deployment types in the future

**Comment by jmeunier28** [comment]
Thanks @drichards-87 ! I think I got everything 👍

**Comment by drichards-87** [comment]
@jmeunier28 Can this be merged now? Or do we need to wait to release this?

**Comment by jmeunier28** [comment]
@drichards-87  yes please merge whenever! thanks!

**Pull request merged**